integration/generator/resources/ec2_linux_test_matrix.json

```diff
@@ -2,14 +2,27 @@
   {
     "os": "ubuntu",
     "username": "ubuntu",
+    "instanceType":"t3a.xlarge",
     "installAgentCommand": "sudo dpkg -i -E ./amazon-cloudwatch-agent.deb",
     "ami": "cloudwatch-agent-integration-test-ubuntu*",
     "caCertPath": "/etc/ssl/certs/ca-certificates.crt",
-    "arc": "amd64", "binaryName": "amazon-cloudwatch-agent.deb"
+    "arc": "amd64",
+    "binaryName": "amazon-cloudwatch-agent.deb"
+  },
+  {
+    "os": "debian-10-arm64",
+    "username": "admin",
+    "instanceType": "c6g.xlarge",
```

Comment:
Why do you choose this instance type?

Reply:
Debian for ARM does not support the default one `t3a.xlarge`. Therefore, I have to choose a similar one with sufficient hardware as possible.